src/components/inline-link/index.tsx

```diff
@@ -12,6 +12,7 @@ import s from './inline-link.module.css'
 const InlineLink = ({
 	children,
 	className,
+	color = 'primary',
```

Comment:
Adding support for the 'secondary' color type that's [documented in HDS. ](https://helios.hashicorp.design/components/link/inline?tab=specifications)